### ISSUE TITLE
Problem creating dataset with 5.0-SNAPSHOT

### ISSUE THREAD

**Issue by SvenDowideit** [Unidata/thredds#977]
I've a simplistic cfg that works with the 4.6.11 docker-compose setup, which fails when switching to 5.0-SNAPSHOT.

```
2017-12-15T00:43:08.833 +0000 [    110403][       2] WARN  - uk.ac.rdg.resc.edal.wms.WmsServlet - Wms Exception caught: "Problem creating dataset fx3/gbr4_rivers_simple_2017-10.nc at /usr/local/tomcat/content/thredds/public/fx3/gbr4_rivers_simple_2017-10.nc" from:uk.ac.rdg.resc.edal.dataset.cdm.CdmDatasetFactory:115
```

I presume this is related to #618 - @rsignell-usgs appears to have much more details in a comment there too (I'm an utter newbie wrt thredds)

**Comment by SvenDowideit** [comment]
I'm going to try building a newer image using https://artifacts.unidata.ucar.edu/#browse/browse/assets:unidata-snapshots just in case its a simple "fixed in dev"

**Comment by lesserwhirls** [comment]
Greetings @SvenDowideit! Thanks for giving this a try in a snapshot version, but be aware that all of 5.0 is dev at this point :-) It's possible that our code in 5.0 is too far out of sync of the `edal-java` version we are using at this point (there is a circular dependency in play here). I've bumped the `edal-java` version locally to see if perhaps updating those libraries would fix the issue, but now I've ran into another issue that needs to be addressed before we can use the latest `edal-java` stuff. I'll keep you posted when another snapshot is ready if you would not mind giving it a try.

**Comment by SvenDowideit** [comment]
awesome :) thanks - because its so easy to switch versions, I thought I'd see what happened - I'm happy to try things any time.

**Comment by lesserwhirls** [comment]
Any chance I could get a copy of your config and a sample file to try?

**Comment by SvenDowideit** [comment]
yup, I'm at a linux.conf.au this week - I'll push something through to you later, or possibly next week.

**Comment by lesserwhirls** [comment]
Great - thank you @SvenDowideit!

**Comment by SvenDowideit** [comment]
I've pushed what I was using to a branch: https://github.com/SvenDowideit/thredds-docker/tree/first-try 

I use the `Makefile` to kick things off - but admittedly, there are some site specific paths that you'll probably need to mod.

The data can be pulled using `https://github.com/SvenDowideit/thredds-docker/blob/first-try/data/update-data.sh` - but I don't think you need very much of it :)

**Comment by lesserwhirls** [comment]
Great - thanks @SvenDowideit! Is it the fx3 or fx4 (or both) that are not working with WMS?

**Comment by SvenDowideit** [comment]
iirc, with v5, it just fails to start.

**Comment by lesserwhirls** [comment]
I think we have this fixed in the latest snapshot of 5.0.
